playbooks/group_vars/webportals.yml

```diff
@@ -333,7 +333,7 @@ webportal_other_config_files:
 skynet_js_org_and_repo: "SkynetLabs/skynet-js"
 skynet_js_repo_url: "https://github.com/{{ skynet_js_org_and_repo }}.git"
 skynet_js_repo_api_url: "https://api.github.com/repos/{{ skynet_js_org_and_repo }}"
-skynet_js_docker_test_image: "node:14-buster-slim"
+skynet_js_docker_test_image: "node:lts-alpine"
```

Comment:
Well yeah I guess, I'll switch it to 18-alpine since skynet-js CI integration tests run on it and it's consistent with what we use for all of our services.

```suggestion
skynet_js_docker_test_image: "node:18-alpine"
```

Just a note, that "14-buster-slim" does not matter since that variable was not unused once we changed how we run integration tests and I probably just missed it at that time.